File: arcade/intro/python/matrixElementsSum.py
def matrixElementsSum(matrix):
    i = 0
    while i < len(matrix):
        i+=1
    nb_of_rows = i
    print("NbofRows:", nb_of_rows)
    nb_columns =0
    for j in matrix[0]:
        nb_columns += 1
    print("nb_colums:", nb_columns)
    for j in range(nb_columns):
        for i in range(nb_of_rows):
            print(matrix[i][j])

File: arcade/intro/C/matrixElementsSum.c
int matrixElementsSum(arr_arr_integer m) {
	int r = 0, i = 0, j;

	for (; i < m.arr[0].size; i++) {
		for(j = 0; j < m.size; j++) {
			if (m.arr[j].arr[i])
				r += m.arr[j].arr[i];
			else break;
		}
	}
	return r;
}

